src/components/results/loadflow/load-flow-result-tab.tsx

```diff
@@ -77,10 +109,76 @@ export const LoadFlowResultTab: FunctionComponent<LoadFlowTabProps> = ({
         filterTab: mappingTabs(tabIndex),
         filterStoreAction: setLoadflowResultFilter,
     });
+    const mapEquipments = useSelector(
+        (state: ReduxState) => state.mapEquipments
+    );
+    const [countriesFilter, setCountriesFilter] = useState<Filter[]>([]);
+    const [voltageLevelsFilter, setVoltageLevelsFilter] = useState<Filter[]>(
+        []
+    );
+    const [globalFilter, setGlobalFilter] = useState<GlobalFilter>();
 
     const { loading: filterEnumsLoading, result: filterEnums } =
         useFetchFiltersEnums(hasFilter, setHasFilter);
 
+    // load countries
+    useEffect(() => {
+        fetchAllCountries(studyUuid, nodeUuid)
+            .then((countryCodes) => {
+                setCountriesFilter(
+                    countryCodes.map((countryCode: string) => ({
+                        label: countryCode,
+                        filterType: 'country',
+                    }))
+                );
+            })
+            .catch((error) => {
+                snackError({
+                    messageTxt: error.message,
+                    headerId: 'FetchCountryError',
+                });
+            });
+    }, [nodeUuid, studyUuid, snackError]);
```

Comment:
Fix :Add mapEquipments dependency
If you create an equipment with a new country the list is not updated

Reply:
Ok, fixed with a dependency on "loadFlowStatus"